.changeset/lemon-masks-carry.md

```diff
@@ -0,0 +1,66 @@
+---
+"@hashicorp/design-system-components": minor
```

Comment:
I see these as fixes, not features. In my view a changelog entry should be a summary, not a log, and should be focused on changes that might affect our consumers (should not include website changes or internal renamings or sorting that don't affect the API)

What about: `Alert`, `Button`, `BadgeCount`, ... ,`Tag`, `Text` – standardize class names and signatures

```suggestion
"@hashicorp/design-system-components": patch
```

Reply:
OK, I've updated the changeset 👍